pages/builds/environment_variables.md.erb

```diff
@@ -4,51 +4,228 @@ When the agent invokes your build scripts it passes in a set of standard Buildki
-  <tr><th><code>BUILDKITE_PULL_REQUEST_BASE_BRANCH</code></th><td>The base branch that the pull request is targetting. <p class="Docs__api-param-eg"><em>Example:</em> <code>"master"</code>, or <code>""</code> if not a pull request.</p></tr>
-  <tr><th><code>BUILDKITE_REBUILT_FROM_BUILD_ID</code></th><td>The UUID of the original build this was rebuilt from. <p class="Docs__api-param-eg"><em>Example:</em> <code>"4735ba57-80d0-46e2-8fa0-b28223a86586"</code>, or <code>""</code> if not a rebuild.</p></td ></tr>
-  <tr><th><code>BUILDKITE_REBUILT_FROM_BUILD_NUMBER</code></th><td>The UUID of the original build this was rebuilt from. <p class="Docs__api-param-eg"><em>Example:</em> <code>"1514"</code>, or <code>""</code> if not a rebuild.</p></td ></tr>
-  </tbody>
+  <tr>
+    <th><code>BUILDKITE</code></th>
+    <td>Always <code>"true"</code>.</td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_AGENT_NAME</code></th>
+    <td>Name of the agent running the job. <p class="Docs__api-param-eg"><em>Example:</em> <code>"my-agent-1"</code></p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_ARTIFACT_PATHS</code></th>
+    <td>The <a href="/docs/builds/artifacts">artifact paths</a> to upload after the job, if any have been specified. <p class="Docs__api-param-eg"><em>Example:</em> <code>"tmp/capybara/**/*;coverage/**/*"</code></p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_BRANCH</code></th>
+    <td>The branch being built. <p class="Docs__api-param-eg"><em>Example:</em> <code>"master"</code></p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_BUILD_CHECKOUT_PATH</code></th>
+    <td>The path where the agent has checked out your code for this build. <p class="Docs__api-param-eg"><em>Example:</em> <code>"/var/lib/buildkite-agent/builds/agent-1/pipeline-2"</code></p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_BUILD_CREATOR</code></th>
+    <td>The name of the user who created the build. <p class="Docs__api-param-eg"><em>Example:</em> <code>"Tracy Tester"</code></p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_BUILD_CREATOR_EMAIL</code></th>
+    <td>The notification email of the user who created the build. <p class="Docs__api-param-eg"><em>Example:</em> <code>"tracy@acme-inc.com"</code></p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_BUILD_CREATOR_TEAMS</code></th>
+    <td>A colon separated list of team names that the build creator belongs to. <p class="Docs__api-param-eg"><em>Example:</em> <code>"everyone:platform"</code></p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_BUILD_ID</code></th>
+    <td>The UUID of the original build this was rebuilt from. <p class="Docs__api-param-eg"><em>Example:</em> <code>"4735ba57-80d0-46e2-8fa0-b28223a86586"</code>, or <code>""</code> if not a rebuild.</p></td>
```

Comment:
Oops, maybe `BUILDKITE_BUILD_ID` and `BUILDKITE_REBUILT_FROM_BUILD_ID` have been mixed up here?

Reply:
Hah, some copy pasta most likely. Should it just be 'The UUID of the build'?